extensions/amp-story/1.0/amp-story.css

```diff
@@ -12,6 +12,13 @@
 @import './pagination-buttons.css';
 
 /** Common */
+:root {
+  --story-page-vw: 1vw;
+  --story-page-vh: 1vh;
+  --story-page-vmin: min(var(--story-page-vw), var(--story-page-vh));
+  --story-page-vmax: max(var(--story-page-vw), var(--story-page-vh));
+  --i-amphtml-story-page-50vw: calc(var(--story-page-vw) * 50);
```

Comment:
The `round()` was really needed, but I think it was for the 3 panels desktop so hopefully we can get rid of this variable entirely.

Reply:
True, it gets removed entirely on #35932 so I'll also remove it from here, wait for that PR to merge, and then sync this branch and ensure everything is ok